src/partial_comparable.cr

```diff
@@ -62,5 +62,18 @@ module PartialComparable(T)
     end
   end
 
+  # The comparison operator.
+  #
+  # Returns a negative number if the object
+  # is considered to be less than *other*, zero if the two objects are equal,
+  # a positive number if this object is considered to be greater than *other*,
+  # or `nil` if no order can be established.
+  #
+  # Subclasses define this method to provide class-specific ordering.
+  #
+  # ```
+  # # Sort in a descending way
```

Comment:
This example is unneccesarily hard to follow. Maybe just prepend the same example with ascending order?